image/tools/lib/component/resources.sh

```diff
@@ -0,0 +1,91 @@
+TS=$(date '+%Y_%m_%d_%H_%M_%S')
+
+function get_middleware_namespaces {
```

Comment:
Could those namespaces be retrieved from an env var (stored in the component secret)? 

This is would make the script it less "specialized" and just cares about having a namespace list

Reply:
An env var would also work, on the other hand this works automatically if new middleware namespaces are added, so i'd prefer this.